tasks/utilities/kenna_api_token_check/kenna_api_token_check.rb

```diff
@@ -18,7 +18,12 @@ def self.metadata
               type: "hostname",
               required: false,
               default: "api.kennasecurity.com",
-              description: "Kenna API Hostname" }
+              description: "Kenna API Hostname" },
+            { name: "show_api_key",
+              type: "flag",
```

Comment:
Why not use the `"boolean"` type?

Reply:
I like strings.

Reply:
That's fine. I've just worked with CLIs where some values were yes/no and some were true/false and it's super confusing, especially when you have to use them together. I'm new here, so I'll not get in the way. 😃

Reply:
No problem.  Thank-you for your review.